From 91bedd280b8a3fb4665db627559abba960be4212 Mon Sep 17 00:00:00 2001
From: Andrew Savchenko <bircoph@gmail.com>
Date: Sat, 5 Sep 2020 14:40:07 +0300
Subject: [PATCH 02/10] Fix configure when /bin/sh is not bash

--Signature=_Sat__5_Sep_2020_14_40_08_+0300_w+XY/NnD8_G.Kd1s
Content-Type: text/plain; charset=US-ASCII
Content-Disposition: inline
Content-Transfer-Encoding: quoted-printable

When /bin/sh used by autoconf is not bash, e.g. dash, configure
fails because it uses bash-specific equality operator "==".

Fix this problem by replacing "==" with POSIX "=" which is
sufficient for test where it is being used.

Upstream-Status: Backport [https://sourceforge.net/p/oprofile/oprofile/ci/91bedd280b8a3fb4665db627559abba960be4212/]
Signed-off-by: Andrew Savchenko <bircoph@gmail.com>
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 05609f6e..f5fcd17d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -466,8 +466,8 @@ AX_COPY_IF_CHANGE(doc/xsl/catalog-1.xml, doc/xsl/catalog.xml)
 
 if ! test "x$enable_account_check" = "xyes"; then
 	:
-elif test "`getent passwd oprofile 2>/dev/null`" == "" || \
-     test "`getent group oprofile 2>/dev/null`" == ""; then
+elif test "`getent passwd oprofile 2>/dev/null`" = "" || \
+     test "`getent group oprofile 2>/dev/null`" = ""; then
 	if test `id -u` != "0"; then
 		echo "Warning: The user account 'oprofile:oprofile' does not exist on the system."
 		echo "         To profile JITed code, this special user account must exist."
-- 
2.31.0

